modules/applications/pages/download-and-launch.adoc

```diff
@@ -1,8 +1,202 @@
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;  # Forwards the X-Forwarded-For header.
+            proxy_set_header X-Forwarded-Proto $scheme;  # Forwards the scheme (http or https) used in the request.
+            proxy_cookie_path /bonita /; # Modifies the cookie path to remove '/bonita'.
+        }
+
+        # Default location block: handles all other requests.
+        location / {
+            auth_request /bonita/API/system/session/1;  # auth_request directive: Makes a subrequest to authenticate the request.
+            error_page 401 = @redirect_to_login;  # Defines a custom error page for 401 Unauthorized responses.
+            proxy_pass http://appsmith;  # Proxies requests to the Appsmith application.
+            access_log /tmp/as_logging.log upstream_logging;  # Logs requests using the custom logging format.
+            proxy_set_header Host $host;  # Forwards the Host header.
+            proxy_set_header X-Real-IP $remote_addr;  # Forwards the real IP address.
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;  # Forwards the X-Forwarded-For header.
+            proxy_set_header X-Forwarded-Proto $scheme;  # Forwards the request scheme.
+            proxy_http_version 1.1;  # Sets the HTTP version used in the proxy request.
+            proxy_set_header Accept-Encoding "";  # Clears the Accept-Encoding header.
+            proxy_set_header X-Forwarded-Host $host;  # Forwards the original host requested.
+            proxy_set_header X-Forwarded-Port $server_port;  # Forwards the server port.
+            proxy_set_header Upgrade $http_upgrade;  # Handles protocols upgrades, e.g., for WebSocket.
+            proxy_set_header Connection "upgrade";  # Maintains upgrade header connections.
+        }
+
+        # Named location for handling redirections when unauthorized access is detected.
+        location @redirect_to_login {
+            return 302 http://localhost/bonita/apps/sendRedirect?redirectUrl=$encoded_uri;  # Redirects to the login page, appending the original requested URL.
+        }
+    }
+}
+----
+====
+
+== Download and launch the docker image
+
+The UI Builder docker image is located in the xref:software-extensibility:bonita-repository-access.adoc[Bonita Artifact Repository].
+Once you got the credentials, you have to authenticate with the `docker login` command:
+[source, shell, subs="+attributes"]
+----
+docker login {bonitasoft-registry}
+----
+
+Run the following command to launch the Bonita UI Builder:
+[source, shell]
+----
+docker compose up -d
+----
+
+Then, you can access the Bonita UI Builder at `http://localhost`.
+
+[NOTE]
+====
+The first time, the initialisation can take a while, a waiting message appears. Reload the page if the page is blank.
+====
+
+== Start Bonita on the local machine
+
+To use the Bonita UI Builder, a Bonita runtime running on your local machine is required (it may be a Bonita Studio or a standalone Bonita runtime).
```

Comment:
```suggestion
To use Bonita UI Builder, a Bonita runtime running on your local machine is required (it may be a Bonita Studio or a standalone Bonita runtime).
```